### ISSUE TITLE
A11y_Azure Data Studio_Azure Added Account_Keyboard: Keyboard focus is not in logical order after pressing tab from "Remove Account' button.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/azuredatastudio#14170]
“[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.3;#keyboard;#FocusOrder;#AINotInScope;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Windows10

**Repro Steps:**
1.Launch Azure Data Studio insiders application.
2.Tab to Connections from left menu and in servers button and hit enter on "New Connection" button.
3.Connections section will get open.
4.Tab to "Browse Preview" Tab item and hit enter.
5.Navigate to "Azure" tree item with the help of Tab key and press Shift+F10.
6.Select "Azure Signin" Menu Item and hit enter on it.
7.Tab to "Add an account" button and add account.
8."AzureAdded Account" screen is visible now.
9.Check keyboard focus order after pressing tab from "remove account" button.

**Actual:**
Keyboard focus is not in logical order after pressing tab from "Remove Account' button.After pressing tab from "Remove Account' button keyboard focus goes to unnecessary pane. 

**Expected:**
Keyboard focus should be in correct logical order.After pressing tab from "Remove Account' button keyboard focus Should go on "Close" button instead of going to "Unnecessary Pane".

**User Impact:**
Keyboard user will get confuse and will be facing problem in navigating the application if focus is not in logical order.

**Comment by Roopesh-T** [comment]
issue is not repro so closing the bug.
![image](https://user-images.githubusercontent.com/86601127/220090447-b4cecc30-8d57-41bd-9c23-ee915742deff.png)
![14170_Closed](https://user-images.githubusercontent.com/86601127/220090516-f1b8db42-6fed-4856-97d1-4beba8479e87.gif)
**Git HUb**:#Closed;
